lib/api/helpers.rb

```diff
@@ -55,6 +55,36 @@ def find_project(id)
       end
     end
 
+    def project_issue
+      begin
+        @issue ||= user_project.issues.find(params[:issue_id])
+      rescue
+        @issue ||= user_project.issues.find_by(iid: params[:issue_id])
+      end
+
+      @issue || not_found!("Issue")
+    end
+
+    def project_merge_request
+      begin
+        @merge_request ||= user_project.merge_requests.find(params[:merge_request_id])
+      rescue
+        @merge_request ||= user_project.merge_requests.find_by(iid: params[:merge_request_id])
+      end
+
+      @merge_request || not_found!("Merge Request")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.